Tests/Microsoft.AppCenter.Test.Windows/Utils/UserIdContextTest.cs

```diff
@@ -14,7 +14,7 @@ public class UserIdContextTest
         [TestInitialize]
         public void InitializeUserIdContextTest()
         {
-           UserIdContext.Instance = new UserIdContext();
+            UserIdContext.Instance = new UserIdContext();
```

Comment:
Is this a mistake?

Reply:
I think this tab is one character short than all other tabs. That's why I corrected it.

Reply:
always use ctrl+k ctrl+c for that, never manually

Reply:
Yes, I did this so it shows that this tab needs more space. Initially I had tests here to test the getter and setter and one reviewer said that's unnecessary so I removed it. But the file formatting changes remained.